index.html

```diff
@@ -34,3 +34,223 @@ <h1 class="homepage-heading">Emmanuel <br>
+                    </div>
+                </div>
+            </div>
+            <div class="projects-project4"></div>
+            <div class="projects-img2">
+                <p class="heading-blue-background">Monty</p>
+                <div class="buttons">
+                    <button class="button1">HTML/CSS</button>
+                    <button class="button2">Ruby on Rails</button>
+                    <button class="button3">JavaScript</button>
+                </div>
+                <div class=rectangle-and-arrow>
+                    <img class="rectangle" src="images/Rectangle20.png" alt="Rectangle">
+                    <div class="union">
+                    <img class="arrow" src="images/Union.png" alt="union">
+                    </div>
+                    <div class="see-project">
+                        <p class="see"><a href="#">see this project</a></p>
+                    </div>
+                </div>
+            </div>
+            <div class="projects-project5"></div>
+            <div class="projects-img2">
+                <p class="heading-blue-background"> Simple Shell Project</p>
+                <div class="buttons">
+                    <button class="button1">HTML/CSS</button>
+                    <button class="button2">Ruby on Rails</button>
+                    <button class="button3">JavaScript</button>
+                </div>
+                <div class=rectangle-and-arrow>
+                    <img class="rectangle" src="images/Rectangle20.png" alt="Rectangle">
+                    <div class="union">
+                    <img class="arrow" src="images/Union.png" alt="union">
+                    </div>
+                    <div class="see-project">
+                        <p class="see"><a href="#">see this project</a></p>
+                    </div>
+                </div>
+            </div>
+            <div class="projects-project6"></div>
+            <div class="projects-img2">
+                <p class="heading-blue-background">Shell Descripts Project</p>
+                <div class="buttons">
+                    <button class="button1">HTML/CSS</button>
+                    <button class="button2">Ruby on Rails</button>
+                    <button class="button3">JavaScript</button>
+                </div>
+                <div class=rectangle-and-arrow>
+                    <img class="rectangle" src="images/Rectangle20.png" alt="Rectangle">
+                    <div class="union">
+                    <img class="arrow" src="images/Union.png" alt="union">
+                    </div>
+                    <div class="see-project">
+                        <p class="see"><a href="#">see this project</a></p>
+                    </div>
+                </div>
+            </div>
+            <section class="About">
+                <div class="about-container">
+                    <div class="about-heading">
+                        <p class="headingg">About</p>
```

Comment:
- [ ] Please consider changing the About title tag to `hx` tag instead of a paragraph 👍🏼